src/Application.vala

```diff
@@ -18,6 +18,9 @@
  * Authored by: Corentin Noël <corentin@elementary.io>
  */
 
+using Soup;
```

Comment:
elementary codestyle is to explicitly namespace. So instead of `using foo`, we make sure to disambiguate with `foo.bar` each time. This is mainly because some libraries use the same class names for different things and it can cause issues when you're unsure if this is a `Gtk.Foo` or ` GLib.Foo` or maybe a `Granite.Foo` or `Clutter.Foo`, etc etc